kivy/base.py

```diff
@@ -203,6 +203,17 @@ def remove_postproc_module(self, mod):
         if mod in self.postproc_modules:
             self.postproc_modules.remove(mod)
 
+    def remove_android_splash(self, *args):
+        '''Remove android presplash in SDL2 bootstrap.'''
+        try:
+            from jnius import autoclass
+        except ImportError:
+            Logger.error('Base: Could not remove android presplash')
+            return
+
+        activity = autoclass('org.kivy.android.PythonActivity').mActivity
```

Comment:
This part should also be wrapped in try/except I think - basically, this call should never be allowed to crash.